data/campaigns/Eastern_Invasion/_main.cfg

```diff
@@ -25,20 +26,18 @@
 [binary_path]
     path=data/campaigns/Eastern_Invasion
 [/binary_path]
-{campaigns/Eastern_Invasion/terrain.cfg}
-{campaigns/Eastern_Invasion/terrain-graphics.cfg}
 #endif
 #ifdef EDITOR
 [binary_path]
-    path=data/campaigns/Eastern_Invasion
+    path=data/resources/Urban_Jungle
```

Comment:
This is unnecessary (and also wrong if EI has any other images in it). Since the resource is loaded, the binary path declared by the resource will be in scope. (Of course, the resource will also need to declare it in an `#ifdef EDITOR` block.) So, just delete this entire block. In fact, you can probably delete the entire `#if EDITOR` section – all of that belongs in the resource, not here in EI.

Reply:
Ok, deleted